linkerd/app/inbound/src/http/mod.rs

```diff
@@ -148,7 +148,11 @@ where
             .push_on_response(http_tracing::client(rt.span_sink.clone(), trace_labels()))
             .push_on_response(http::BoxResponse::layer())
             .check_new_service::<Target, http::Request<_>>();
-
+        let no_profile = target
+            .clone()
+            .push_on_response(http::BoxResponse::layer())
+            .check_new_service::<Target, http::Request<_>>()
+            .into_inner();
```

Comment:
nit: vertical whitespace is easier on the eyes

```suggestion

        let no_profile = target
            .clone()
            .push_on_response(http::BoxResponse::layer())
            .check_new_service::<Target, http::Request<_>>()
            .into_inner();
            
```